.gitlab-ci.yml

```diff
@@ -36,8 +36,8 @@ variables:
   # The $hash is the first 10 characters of the md5 of the Dockerfile. e.g.
   # echo $(md5sum dev/ci/docker/old_ubuntu_lts/Dockerfile | head -c 10)
   # echo $(md5sum dev/ci/docker/edge_ubuntu/Dockerfile | head -c 10)
-  BASE_CACHEKEY: "old_ubuntu_lts-V2024-01-08-011994e15c"
-  EDGE_CACHEKEY: "edge_ubuntu-V2024-01-08-6b5ac95a11"
+  BASE_CACHEKEY: "old_ubuntu_lts-V2023-11-30-7323af070e"
+  EDGE_CACHEKEY: "edge_ubuntu-V2023-12-29-20017bf137"
```

Comment:
maybe worth updating the dates here
and why is the base cachekey changing, you only touched the edge dockerfile?